subgraph-js/src/__tests__/buildSubgraphSchema.test.ts

```diff
@@ -518,6 +518,50 @@ extend type User @key(fields: "email") {
 }
 `);
     });
+
+    describe('@tag directive', () => {
+      const query = `query GetServiceDetails {
+        _service {
+          sdl
+        }
+      }`;
+
+      const validateTag = async (header: string) => {
+        const schema = buildSubgraphSchema(gql`${header}
+          type User @key(fields: "email") @tag(name: "tagOnType") {
+            email: String @tag(name: "tagOnField")
+          }
+
+          interface Thing @tag(name: "tagOnInterface") {
+            name: String
+          }
+
+          union UserButAUnion @tag(name: "tagOnUnion") = User
+        `);
+
+        const { data, errors } = await graphql({ schema, source: query });
+        expect(errors).toBeUndefined();
+        expect((data?._service as any).sdl).toEqual(`${header}type User @key(fields: "email") @tag(name: "tagOnType") {
+  email: String @tag(name: "tagOnField")
+}
+
+interface Thing @tag(name: "tagOnInterface") {
+  name: String
+}
+
+union UserButAUnion @tag(name: "tagOnUnion") = User
+`);
+      };
+
+      it('adds it for fed1 schema', async () => {
```

Comment:
*nitpick/question* We're saying that the tag directive is not added for fed1, right?